### ISSUE TITLE
Add. 17,253, foll. 70–103 | https://bl.vuexistapps.us/manuscript/1047 (Part2)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1032]
### Problems with Manuscript Overview

- Date is missing from the MS overview. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- "foLl." should be corrected to "foll." in "(Add. 17,253, foLl. 70—103)"
- "foil." should be corrected to "foll." in "especially foil. 70, 75"
- Hand 1: "xiii cent." should be changed to "xiiith cent."
- Perhaps Hand 2 should also be added, based on the sentence "and by later hands from ܟܒ to ܟܗ and from ܝܚ to ܟܐ."

### Categories of Physical Description Problems

- [ ] Whitespace
- [X] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
Resolved: all typos fixed

Question: Do we want a separate hand for the letters signed by a later hand?
@wlpotter

**Comment by wlpotter** [comment]
Yes, let's add a hand for this. We can keep the sentence in the phys desc as-is (just to have all that relevant info in one spot). The hand note can say something to the effect "Later hands supplied additional quire signatures"

**Comment by mikey-tang16** [comment]
Hand2 added
